validation-test/compiler_crashers_2_fixed/0153-rdar39130543.swift

```diff
@@ -1,5 +1,5 @@
-// RUN: not --crash %target-swift-frontend %s -emit-ir
-// REQUIRES: asserts
+// RUN: %target-swift-frontend %s -emit-ir
```

Comment:
This is not-crashing for the wrong reasons still :)

Reply:
Ahh, right. This one. Thanks for the reminder